apix-rest-v1/src/main/java/eu/xenit/apix/rest/CheckTomcatConfiguration.java

```diff
@@ -0,0 +1,60 @@
+package eu.xenit.apix.rest;
+
+import java.io.BufferedReader;
+import java.io.InputStreamReader;
+import java.nio.charset.StandardCharsets;
+import java.util.stream.Collectors;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.core.io.FileSystemResource;
+import org.springframework.core.io.Resource;
+
+public class CheckTomcatConfiguration {
+    private static final Logger logger = LoggerFactory.getLogger(CheckTomcatConfiguration.class);
+
+    public CheckTomcatConfiguration() {
+        readTomcatContextFile();
+    }
+
+    public void readTomcatContextFile(){
+        try {
+        // TODO: tomcat-embedded rmi/JNDI is broken
+        // Set RMI ON with tomcat embedded, but no JNDI is possible...
+//            Context ctx = new InitialContext();
+//            String value = (String) ctx.lookup("java:comp/env/myConfig");
+//            logger.error("JNDI ctx.lookup(\"java:comp/env/env/myConfig\"): ", value);
+
+            Resource resource = new FileSystemResource("/usr/local/tomcat/webapps/alfresco/META-INF/context.xml");
```

Comment:
Set via variable in global properties now. Will give a warning message this is not set. 
Can only check if fysical file has the configuration. Will warn that they have to check tomcat config in case of RequestPart || forward-slashes issues.